pandas/io/formats/style.py

```diff
@@ -1511,7 +1511,10 @@ def from_custom_template(cls, searchpath, name):
         """
         loader = jinja2.ChoiceLoader([jinja2.FileSystemLoader(searchpath), cls.loader])
 
-        class MyStyler(cls):
+        # mypy doesnt like dynamically-defined class
+        # error: Variable "cls" is not valid as a type  [valid-type]
+        # error: Invalid base class "cls"  [misc]
+        class MyStyler(cls):  # type:ignore[valid-type,misc]
```

Comment:
does this resolve if using an appropriate constructor like: https://docs.python.org/3.7/library/types.html (or the old type constructor)

Reply:
i didnt know that `types.new_class` was a thing.  id rather keep the more-idiomatic usage even if mypy doesnt like it

Reply:
ok